src/rard/templates/research/add_appositum_fragment_link.html

```diff
@@ -5,20 +5,23 @@
     {% blocktrans with name=form.instance.linked_field %}
         Add appositum link to fragment
     {% endblocktrans %}
+    {{ form.media.css }}
 {% endblock %}
 
 {% block inner %}
 
     <form novalidate enctype="multipart/form-data" autocomplete='off' action='{{ request.path }}' class="form" method='POST'>
         {% csrf_token %}
+        <input type="text" id="search-fragments" name="search-fragments" placeholder="Start typing to search for fragments..." hx-get="{% url 'fragment:fetch_fragments' %}" hx-trigger="keyup delay:500ms changed" hx-swap="innerHTML" hx-target="#search-results" class="form-control">
```

Comment:
This element seems a bit close to the one below, could maybe do with a bigger margin?